app/assets/stylesheets/admin/calendar/_calendar.scss

```diff
@@ -90,7 +90,7 @@
   }
 
   .loan-calendar {
-    .cal-step {
+    .cal-step, td {
       &:hover {
```

Comment:
I don't think you need `&:hover` here. `cursor` means on hover by default, doesn't it?